cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationOperations.java

```diff
@@ -0,0 +1,57 @@
+package cloud.foundry.cli.operations;
+
+import cloud.foundry.cli.crosscutting.beans.ApplicationBean;
+import cloud.foundry.cli.crosscutting.beans.Bean;
+import org.cloudfoundry.operations.DefaultCloudFoundryOperations;
+import org.cloudfoundry.operations.applications.ApplicationManifest;
+import org.cloudfoundry.operations.applications.ApplicationSummary;
+import org.cloudfoundry.operations.applications.GetApplicationManifestRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApplicationOperations extends AbstractOperations<DefaultCloudFoundryOperations> {
```

Comment:
Docstring missing. Reading the code, I have no idea what exactly the empty methods are all there for. Please add more documentation.